scripts/vestedWalletsDeployer.ts

```diff
@@ -41,16 +76,60 @@ export const deployVestedWallets = async (
         wallet: w.vestingWalletAddress,
       }));
 
+    // store wallet addresses
     for (const walletPair of walletPairs) {
-      const amount = recipients[walletPair.beneficiary.toLowerCase()];
-
-      if (!amount.gt(0)) {
-        throw new Error(`Missing amount for ${walletPair.beneficiary}`);
-      }
-
-      await (await token.transfer(walletPair.wallet, amount)).wait();
+      data.wallets.push({ beneficiary: walletPair.beneficiary, walletAddress: walletPair.wallet });
     }
+    updateDeployedWallets(deployedWalletsFileLocation, data);
   }
 
   return vestedWalletFactory;
 };
+
+async function main() {
+  console.log("Start vested wallets deployment process...");
+
+  /// get env vars
+  const arbKey = process.env["ARB_KEY"] as string;
+  const arbRpc = process.env["ARB_URL"] as string;
+  const vestedRecipientsLocation = process.env["VESTED_RECIPIENTS_FILE_LOCATION"] as string;
+  const deployedWalletsLocation = process.env["DEPLOYED_WALLETS_FILE_LOCATION"] as string;
+  if (
+    arbKey === undefined ||
+    arbRpc === undefined ||
+    vestedRecipientsLocation === undefined ||
+    deployedWalletsLocation === undefined
+  ) {
+    throw new Error(
+      "Following env vars have to be defined: ARB_KEY, ARB_URL, VESTED_RECIPIENTS_FILE_LOCATION and DEPLOYED_WALLETS_FILE_LOCATION"
+    );
+  }
+
+  const deployedWalletsFileLocation = path.join(__dirname, "..", deployedWalletsLocation);
+  const vestedRecipientsFileLocation = path.join(__dirname, "..", vestedRecipientsLocation);
+  const vestedRecipients = loadRecipients(vestedRecipientsFileLocation);
```

Comment:
how does this handle many wallets w/ the same beneficiary?

Reply:
do we just run this more than once w/ two different recipient jsons?

Reply:
hm yeah that would do it, although it would deploy 2 factories. Probably better if I modify data structure so each beneficiary has array of wallets instead of single one

Reply:
done, updated PR description with new format for input and output